### PR TITLE
Bug 1179896 - Add a switch to enable/disable dom.serviceWorkers.inter…

### PR DESCRIPTION
**Pull request by ferjm**
…ception.enabled and dom.serviceWorkers.testing prefs. r=jaoo

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=2673357a1e56f492fb84791b336df272fa5b506d

File: apps/settings/elements/developer_service_workers.html [hunk 1]
```diff
@@ -9,8 +9,27 @@
       <h1>Service Workers</h1>
     </gaia-header>
 
+
```
Comment (jaoo) [inline apps/settings/elements/developer_service_workers.html]: nit: remove that line please.

File: apps/settings/locales/settings.en-US.properties [hunk 2]
```diff
@@ -1519,6 +1519,10 @@ achievements-sharing-is-caring-description=Share your apps and add-ons with the
 achievements-creative-kind-name=Creative Kind
 achievements-creative-kind-description=Use Theme Editor to create a new theme
 
+# Service Workers
+service-workers-interception=Intercept requests
+service-workers-testing=Allow HTTP
```
Comment (jaoo) [inline apps/settings/locales/settings.en-US.properties]: nit: s/Allow HTTP/Allow over HTTP

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=a051c4a42bcc76291cd38b147a51e4ca393003a2

**Pull request merged**